### PR TITLE
remove all console logs and unused variables

### PR DESCRIPTION
**Pull request by ZhraaBaharAluloom**
## What?
In this PR, removed all unnecessary  console logs and unused variable
## Why?
To deploy the project with no warning and errors

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -78,7 +78,7 @@ function App() {
     return unsubscribe;
   }, [dispatch]);
 
-  // if (isLoading) return <Loader />;
+  if (isLoading) return <Loader />;
```
Comment (RawandDev) [inline src/App.js]: Can we please remove this completely since this displays an ugly loader.

File: src/store/counter/userSlice.js [hunk 2]
```diff
@@ -84,11 +77,12 @@ export const signIn = createAsyncThunk(
...
-        data.setErrors(`Failed to sign in ${error.message}`);
-        const { code } = error;
         const { message } = error;
-        const { email } = error;
-        console.log(code, message, email);
+        message === "Firebase: Error (auth/user-not-found)."
+          ? data.setErrors(`user not found`)
+          : message === "Firebase: Error (auth/wrong-password)."
+          ? data.setErrors(`Wrong Password`)
+          : data.setErrors(``);
```
Comment (RawandDev) [inline src/store/counter/userSlice.js]: This supposed to show a message but doesn't work for me when I login with a wrong password or an account that doesn't work.

**Review by RawandDev — COMMENTED** [review]
Nice work Zahra. I see that the branch name is `fix-deployment-errors` and I expect that this branch should be deployed like the very first PRs we created but it doesn't show any sign for deployment.

File: src/store/counter/userSlice.js (hunk 2 above)
Comment (ZhraaBaharAluloom) [reply src/store/counter/userSlice.js]: done

**Review by RawandDev — APPROVED** [review]
LGTM

**Pull request merged**